src/components/TeamLocations/TeamLocations.jsx

```diff
@@ -0,0 +1,317 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { MapContainer, TileLayer, CircleMarker, Popup } from 'react-leaflet';
+import MarkerClusterGroup from 'react-leaflet-cluster';
+import 'leaflet/dist/leaflet.css';
+import './TeamLocations.css';
+import { Button, Container } from 'reactstrap';
+
+import { boxStyle } from 'styles';
+import { toast } from 'react-toastify';
+import { SEARCH } from 'languages/en/ui';
+import { useDispatch, useSelector } from 'react-redux';
+import { ApiEndpoint, ENDPOINTS } from '../../utils/URL';
+import ListUsersPopUp from './ListUsersPopUp';
+import AddOrEditPopup from './AddOrEditPopup';
+import { getTimeZoneAPIKey } from 'actions/timezoneAPIActions';
+
+function TeamLocations() {
+  const [userProfiles, setUserProfiles] = useState([]);
+  const [manuallyAddedProfiles, setManuallyAddedProfiles] = useState([]);
+  const [addNewIsOpen, setAddNewIsOpen] = useState(false);
+  const [listIsOpen, setListIsOpen] = useState(false);
+  const [editIsOpen, setEditIsOpen] = useState(false);
+  const [editingUser, setEditingUser] = useState(null);
+  const [searchText, setSearchText] = useState('');
+  const role = useSelector(state => state.userProfile.role);
```

Comment:
Change this to: 
`` const role = useSelector(state => state.auth.user.role);``

Reply:
It is a good one! 
Thank you!